set.mm

```diff
@@ -501775,6 +501788,75 @@ have become an indirect lemma of the theorem in question (i.e. a lemma
       WKWKWKWLCWRIXOXHCBWMEWNCWRFGHWOWPWQ $.
   $}
 
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Acyclic graphs
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+  $c AcyclicGraph $.
+
+  $( Extend class notation with acyclic graphs. $)
+  cacycgr $a class AcyclicGraph $.
+
+  ${
+    $d g f p $.
+    $( Define the class of all acyclic graphs.  A graph is called _acyclic_ if
+       it has no (non-trivial) cycles.  (Contributed by BTernaryTau,
+       11-Oct-2023.) $)
+    df-acycgr $a |- AcyclicGraph =
+              { g | -. E. f E. p ( f ( Cycles ` g ) p /\ f =/= (/) ) } $.
+  $}
+
+  ${
+    $d f g p $.
+    $( An alternate definition of the class of all acyclic graphs that requires
+       all cycles to be trivial.  (Contributed by BTernaryTau, 11-Oct-2023.) $)
+    dfacycgr1 $p |- AcyclicGraph =
+              { g | A. f A. p ( f ( Cycles ` g ) p -> f = (/) ) } $=
+      ( cacycgr cv ccycls cfv wbr c0 wne wa wex cab wceq wal df-acycgr 2exanali
+      wn wi df-ne anbi2i 2exbii xchnxbir abbii eqtri ) DAEZCEBEFGHZUFIJZKZCLALZ
+      RZBMUGUFINZSCOAOZBMABCPUKUMBUGULRZKZCLALUMUJUGULACQUIUOACUHUNUGUFITUAUBUC
+      UDUE $.
+  $}
```

Comment:
These two theorems have the same distinct variable conditions, so maybe they could share the same `${`..`$}` scope.

Reply:
I didn't realize this was thing. I've made the change for both pairs that share distinct variable conditions.

Reply:
Thanks!
It's not mandatory and not enforced, it sometimes just helps to group things which belong together, and it's a bit more compact. Usually we do that more for actual `$e` hypotheses.